compiler-core/src/build/project_compiler.rs

```diff
@@ -112,6 +115,11 @@ where
 
     /// Returns the compiled information from the root package
     pub fn compile(&mut self) -> Result<Package> {
+        if cfg!(not(target_arch = "wasm32")) {
```

Comment:
Doesn't look like this does anything might be worth just removing?

Reply:
Never mind this is now in conjunction with a line in `cargo.toml`

Reply:
Would bringing this back help with wasm compilation?

Reply:
Doesn't look like it. Appears like with all the cfg stuff make it so the package doesn't get compiled but it then complains that stuff doesn't exist as using that if doesn't stop it compiling that code as its a runtime check. Is there a way to-do a compile time check @lpil?

Reply:
Oh sorry, I see what you mean now. Got it